release-notes.txt

```diff
@@ -1982,8 +1982,8 @@ API Changes
 - static inline int ipv6_addr_split_iface(char *addr_str)
 + static inline char *ipv6_addr_split_iface(char *addr_str)
 
-- int ipv6_addr_split(char *addr_str, char seperator, int _default);
-+ int ipv6_addr_split_int(char *addr_str, char seperator, int _default);
+- int ipv6_addr_split(char *addr_str, char separator, int _default);
```

Comment:
Should we do this for release notes?

Reply:
TBH, I wasn't sure and even thought of excluding this file from codespell check. But I think it's important to also process this file.
For this particular change, I checked the API and the parameter is `separator` in RIOT. So the typo was fixed in the code already.

Reply:
Ok!

Reply:
> But I think it's important to also process this file.

I think its important, I'm just unsure if we should do it for passed release notes.

Reply:
> I'm just unsure if we should do it for passed release notes

I think the same but it's technically difficult, codespell can only check a whole file or ignore it.

Reply:
Lets get some other input on this, I personally don't mind doing it though.